Description: Make sure that no locales are set, even if the gsettings "region" key has
 been assigned a value in the past, since they are set in Ubuntu via ~/.pam_environment.
Forwarded: not-needed
Author: Gunnar Hjalmarsson <ubuntu@gunnar.cc>

*
* The set_locale() function is questionable anyhow, since it's called after ~/.profile
* is sourced by e.g. lightdm or gdm, which prevents users from fine tuning the locale
* settings by editing ~/.profile.
*

Index: gnome-settings-daemon-3.6.1/gnome-settings-daemon/main.c
===================================================================
--- gnome-settings-daemon-3.6.1.orig/gnome-settings-daemon/main.c	2012-10-05 14:41:44.000000000 +0200
+++ gnome-settings-daemon-3.6.1/gnome-settings-daemon/main.c	2012-10-26 10:20:14.163005369 +0200
@@ -316,9 +316,6 @@
                 g_debug ("Could not connect to the Session manager: %s", error->message);
                 g_error_free (error);
         } else {
-                /* Always call this first, as Setenv can only be called before
-                   any client registers */
-                set_locale (proxy);
 #ifdef HAVE_IBUS
                 /* This will register with gnome-session after calling Setenv. */
                 set_legacy_ibus_env_vars (proxy);
